src/VkBootstrap.h

```diff
@@ -121,7 +121,10 @@ struct GenericFeaturesPNextNode {
 
 	GenericFeaturesPNextNode();
 
-	template <typename T> void set(T const& features) { *reinterpret_cast<T*>(this) = features; }
+	template <typename T>
+	GenericFeaturesPNextNode(T const& features) {
+		*reinterpret_cast<T*>(this) = features;
+	}
 
 	static bool match(GenericFeaturesPNextNode const& requested, GenericFeaturesPNextNode const& supported);
```

Comment:
might as well ask for noexcept on eveything...
But the whole code file could probably get that treatment, so feel free to not add noexcept to the changed code, just say so (that way a single 'noexcept everything' can be made)